support-frontend/assets/pages/digital-subscription-landing/components/paymentSelection/helpers/paymentSelection.jsx

```diff
@@ -31,21 +31,41 @@ const BILLING_PERIOD = {
   [Monthly]: {
     title: 'Monthly',
     singlePeriod: 'month',
-    salesCopy: (displayPrice: string, saving?: string) => (
-      <span>14 day free trial, then <strong>{displayPrice}</strong> a month for 12 months {saving && null}
-        <br className="product-option__full-screen-break" />
-      </span>
+    salesCopy: (displayPrice: string, saving?: string) => ({
+      control: () => (
+        <span>14 day free trial, then <strong>{displayPrice}</strong> {saving && null}
+          <br className="product-option__full-screen-break" />
+          <br className="product-option__full-screen-break" />
```

Comment:
why do we need two line breaks when not on mobile breakpoint? presume it looks ok even at the different sizes but I was just interested